src/utils/Utils.re

```diff
@@ -62,50 +62,50 @@ let getFields = (~entity, ~kind=?, ()): array(Type.field) => {
         {name: "operation_group_hash", displayName: "Operation Hash", isLink: false},
         {name: "kind", displayName: "Kind", isLink: false},
         {name: "block_hash", displayName: "Block Hash", isLink: true},
-        {name: "source", displayName: "From", isLink: false},
-        {name: "destination", displayName: "To", isLink: false},
+        {name: "source", displayName: "From", isLink: true},
```

Comment:
None of this should be hard-wired, it should be generated from the metadata service.